Missing runtime assemblies on Debian Stretch

The famous issue #204 fails on Debian Stretch

```
root@70144b5a5421:/dotnet-script/src/Dotnet.Script.Tests# dotnet test
Build started, please wait...
Build completed.

Test run for /dotnet-script/src/Dotnet.Script.Tests/bin/Debug/netcoreapp2.0/Dotnet.Script.Tests.dll(.NETCoreApp,Version=v2.0)
Microsoft (R) Test Execution Command Line Tool Version 15.6.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Starting test execution, please wait...
[xUnit.net 00:00:00.7570662]   Discovering: Dotnet.Script.Tests
[xUnit.net 00:00:00.8246965]   Discovered:  Dotnet.Script.Tests
[xUnit.net 00:00:00.8317680]   Starting:    Dotnet.Script.Tests
[xUnit.net 00:01:44.5950436]     Dotnet.Script.Tests.ScriptExecutionTests.ShouldHandleIssue204 [FAIL]
[xUnit.net 00:01:44.5990250]       Assert.Contains() Failure
[xUnit.net 00:01:44.5995086]       Not found: System.Net.WebProxy
[xUnit.net 00:01:44.5997346]       In value:  /dotnet-script/src/Dotnet.Script.Tests/bin/Debug/netcoreapp2.0/../../../TestFixtures/Issue204/Issue204.csx(5,26): error CS0246: The type or namespace name 'WebProxy' could not be found (are you missing a using directive or an assembly reference?)
[xUnit.net 00:01:44.6009356]       Stack Trace:
[xUnit.net 00:01:44.6021430]         /dotnet-script/src/Dotnet.Script.Tests/ScriptExecutionTests.cs(132,0): at Dotnet.Script.Tests.ScriptExecutionTests.ShouldHandleIssue204()
Failed   Dotnet.Script.Tests.ScriptExecutionTests.ShouldHandleIssue204
Error Message:
 Assert.Contains() Failure
Not found: System.Net.WebProxy
In value:  /dotnet-script/src/Dotnet.Script.Tests/bin/Debug/netcoreapp2.0/../../../TestFixtures/Issue204/Issue204.csx(5,26): error CS0246: The type or namespace name 'WebProxy' could not be found (are you missing a using directive or an assembly reference?)
Stack Trace:
   at Dotnet.Script.Tests.ScriptExecutionTests.ShouldHandleIssue204() in /dotnet-script/src/Dotnet.Script.Tests/ScriptExecutionTests.cs:line 132
[xUnit.net 00:02:36.5310156]   Finished:    Dotnet.Script.Tests

Total tests: 58. Passed: 57. Failed: 1. Skipped: 0.
Test Run Failed.
Test execution time: 2.6220 Minutes
root@70144b5a5421:/dotnet-script/src/Dotnet.Script.Tests# 
```

This is the root cause behind the broken build reported in #252.
The identifier in this case is `debian.9-x64`.

Repro
1. Install Docker
2. navigate to `dotnet-script` folder
3. `sudo docker run -it --rm -v $(pwd):/dotnet-script microsoft/dotnet:2.0-sdk /bin/bash`
4. you are now inside the bash of the docker container. `cd dotnet-script`
5. `dotnet build`
6. `dotnet test src/Dotnet.Script.Tests/Dotnet.Script.Tests.csproj`

It could be that we don't load dependencies from the assets file correctly.

--- a/src/Dotnet.Script.DependencyModel/Environment/RuntimeHelper.cs
+++ b/src/Dotnet.Script.DependencyModel/Environment/RuntimeHelper.cs
@@ -52,13 +52,14 @@ namespace Dotnet.Script.DependencyModel.Environment
         }
 
         public static string GetRuntimeIdentifier()
-        {            
+        {
             var platformIdentifier = GetPlatformIdentifier();
-            if (platformIdentifier == "osx")
+            if (platformIdentifier == "osx" || platformIdentifier == "linux")
             {
                 return $"{platformIdentifier}-{GetProcessArchitecture()}";
             }
-            return RuntimeEnvironment.GetRuntimeIdentifier();            
+            var runtimeIdentifier = RuntimeEnvironment.GetRuntimeIdentifier();
+            return runtimeIdentifier;
         }
 
         public static string CreateTempFolder(string targetDirectory)

